symfony/client/src/orangehrmAuthenticationPlugin/pages/Login.vue

```diff
@@ -119,6 +126,12 @@ export default {
       this.$refs.loginForm.$el.submit();
     },
   },
+
+  beforeMount() {
+    setTimeout(() => {
+      reloadPage();
+    }, 1200000); // 20 * 60 * 1000 (20 minutes);
+  },
```

Comment:
Added this to address;
https://github.com/orangehrm/orangehrm/commit/a9da037055e5399edbbfadfdaa548b63d24937c6#diff-f51a7389083b23ea24d1c5dac16a022305099f4911532b98ad27da3df252a4ed